inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -428,9 +431,25 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 		}
 	)
 
-	createEffect(() => {
-		if (localStorage?.user?.isLoggedIn) refetchRepoInfo()
-	})
+	const [previousLoginStatus, setPreviousLoginStatus] = createSignal(localStorage?.user?.isLoggedIn)
+	createEffect(
+		on(
```

Comment:
@NiklasBuchfink  i if it works like intended this is fine, but i cannot say i fully understand this :D eg. how previousloginstate knows the previous value after reload and how the signal/on exactly work.